packages/run-protocol/src/psm/psm.js

```diff
@@ -0,0 +1,187 @@
+// @ts-check
+import '@agoric/zoe/exported.js';
+import '@agoric/zoe/src/contracts/exported.js';
+import '@agoric/governance/src/exported';
+import {
+  assertProposalShape,
+  ceilMultiplyBy,
+  floorDivideBy,
+  floorMultiplyBy,
+  makeRatio,
+} from '@agoric/zoe/src/contractSupport/index.js';
+import { Far } from '@endo/marshal';
+// import { CONTRACT_ELECTORATE } from '@agoric/governance';
+
+import { AmountMath } from '@agoric/ertp';
+import { makeMakeCollectFeesInvitation } from '../collectFees.js';
+
+const { details: X } = assert;
+
+/**
+ * @file The Parity Stability Module supports efficiently minting/burning a
+ * stable token at a specified fixed ratio to a reference stable token, which
+ * thereby acts as an anchor to provide additional stability. For flexible
+ * economic policies, the fee percentage for trading into and out of the stable
+ * token are specified separately.
+ *
+ */
+
+const BASIS_POINTS = 10000n;
+
+/**
+ * Stage a transfer of a singe asset from one seat to another, with an optional
+ * remapping of the Keywords. Check that the remapping is for the same amount.
+ *
+ * @param {ZCFSeat} from
+ * @param {ZCFSeat} to
+ * @param {AmountKeywordRecord} txFrom
+ * @param {AmountKeywordRecord} txTo
```

Comment:
It's for internal usage as a sanity check, so I wasn't trying to be too rigorous. It's more useful to have the same format as the arguments to decrementBy et al.  Since those APIs will be changing, I thought it not worth further effort yet.